docs/Guardian User Privileges Diagram.md

```diff
@@ -0,0 +1,23 @@
+# Guardian User Privileges Diagram
+
+
+| PERMISSION AREA                        | ADMINISTRATOR | NURSE | CARETAKER |
+|----------------------------------------|:---------------:|:-------:|:-----------:|
+| Manage user accounts/profiles                   | x             | -     | -         |
+| Configure system settings              | x             | -     | -         |
+| Generate reports                       | x             | -     | -         |
+| Manage application roles and access    | x             | -     | -         |
+| View nursing rosters                   | x             | -     | -         |
+| Communicate with patients              | -             | x     | -         |
+| Record vital signs and measurements    | -             | x     | x         |
+| Access, add and edit tasks             | -             | x     | x         |
+| View patient health records            | -             | x     | x<sup>1</sup>        |
+| Update patient health records          | -             | x     | x<sup>1</sup>       |
+| Receive health alerts                  | -             | -     | x         |
+| Upload care plan                       | -             | -     | x         |
+| View daily report                      | -             | -     | x         |
+
```

Comment:
This table will be really useful to guide development moving forward. One thing that might not be documented, I think probably Admins should be able to create, edit and delete patient profiles. However, nurses should be able to update patient health data

Reply:
Changes made.